### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by tst-swanson**
Updated index at docsrc root to point to main pages for each section

### CODE REVIEW

**Comment by tst-swanson** [comment]
Updated to point to releaseNotes index

File: docs/geedocs/docsrc/index.rst [hunk 1]
```diff
@@ -12,27 +12,27 @@ Google Earth Enterprise Documentation Version 5.3.2
...
 
-   ./answer/3468004
+   ./answer/geeServerAdmin
 
    ./answer/geeServerConfigAndSecurity
 
    ./answer/googleEarthEnterpriseClient
 
-   ./answer/4580164
+   ./answer/welcomePortable5_0
```
Comment (tst-hrajamoney) [inline docs/geedocs/docsrc/index.rst]: this should be portable

File: docs/geedocs/docsrc/index.rst [hunk 2]
```diff
@@ -12,27 +12,27 @@ Google Earth Enterprise Documentation Version 5.3.2
 
    ./answer/fusionTutorial
 
-   ./answer/3468016
+   ./answer/installOverviewGEE
```
Comment (tst-hrajamoney) [inline docs/geedocs/docsrc/index.rst]: this should be installGEE

File: docs/geedocs/docsrc/index.rst [hunk 3]
```diff
@@ -12,27 +12,27 @@ Google Earth Enterprise Documentation Version 5.3.2
...
 
    ./answer/googleEarthEnterpriseClient
 
-   ./answer/4580164
+   ./answer/welcomePortable5_0
 
    ./answer/developers
 
-   ./answer/6121524
+   ./answer/geecheckTestServerStatus
```
Comment (tst-hrajamoney) [inline docs/geedocs/docsrc/index.rst]: this should be "troubleshooting"

**Comment by tst-swanson** [comment]
Updated index to point to correct files and changed base to master

**Pull request closed without merging**